it/firefox-ios.xliff

```diff
@@ -341,7 +344,6 @@
       </trans-unit>
       <trans-unit id="DefaultBrowserCard.Button.v2" xml:space="preserve">
         <source>Learn How</source>
-        <target>Scopri come fare</target>
```

Comment:
This is an example of a string that was removed by this PR (there were 18, IIRC). Could you please investigate why this happened? I don't see corresponding changes in `en-US` or `templates`.  It's also unclear how this PR was created a this point, since it's one commit (it used to be one commit per locale).